invenio_records_marc21/ui/theme/__init__.py

```diff
@@ -26,10 +26,23 @@ def init_theme_views(blueprint, app):
     routes = app.config.get("INVENIO_MARC21_UI_THEME_ENDPOINTS")
 
     blueprint.add_url_rule(routes["index"], view_func=index)
+
+    blueprint.add_url_rule(routes["uploads-marc21"], view_func=uploads_marc21)
     blueprint.add_url_rule(routes["record-search"], view_func=search)
     blueprint.add_url_rule(routes["deposit-create"], view_func=deposit_create)
     blueprint.add_url_rule(routes["deposit-edit"], view_func=deposit_edit)
 
+    @blueprint.before_app_first_request
```

Comment:
Add a TODO to fix this after https://github.com/inveniosoftware/invenio-base/issues/171 has been solved

Reply:
A fix me note was added